libs/island-ui/core/src/lib/ActionCategoryCard/ActionCategoryCard.tsx

```diff
@@ -0,0 +1,323 @@
+import React, { ReactElement, ReactNode, forwardRef, ElementType } from 'react'
+import cn from 'classnames'
+import { ObjectFitProperty } from 'csstype'
+import { useMeasure } from 'react-use'
+import { UseMeasureRef } from 'react-use/lib/useMeasure'
+
+import { Box } from '../Box/Box'
+import { FocusableBox } from '../FocusableBox/FocusableBox'
+import { Inline } from '../Inline/Inline'
+import { Tag, TagProps } from '../Tag/Tag'
+import { Hyphen } from '../Hyphen/Hyphen'
+import { Text, TextProps } from '../Text/Text'
+
+import * as styles from './ActionCategoryCard.css'
+import { Checkbox } from '../Checkbox/Checkbox'
+import { Button, ButtonSizes, ButtonTypes } from '../Button/Button'
+import { Icon as IconType } from '../IconRC/iconMap'
+import { LinkV2 } from '../Link/LinkV2'
+
+export const STACK_WIDTH = 280
+
+export type CategoryCardTag = {
+  label: string
+  href?: string
+  onClick?: () => void
+  disabled?: boolean
+  outlined?: boolean
+}
+
+export type CTAProps = {
+  label: string
+  /** Allows for simple variant configuration of the button. If buttonType is defined it will supersede this property. */
+  variant?: ButtonTypes['variant']
+  /** Allows for full buttonType control. Supersedes the variant property when both are defined. */
+  buttonType?: ButtonTypes
+  size?: ButtonSizes
+  icon?: IconType
+  iconType?: 'filled' | 'outline'
+  onClick?: () => void
+  disabled?: boolean
+}
+
+type SidePanelItems = {
+  icon?: React.ReactNode
+  title: string
+}
+
+type SidePanelConfigProps = {
+  buttonLabel?: string
+  items?: Array<SidePanelItems>
+  cta?: CTAProps
+}
+
+export type ActionCategoryCardProps = {
+  ref?: UseMeasureRef<HTMLElement>
+  width?: number
+  icon?: React.ReactElement
+  heading: string
+  headingAs?: TextProps['as']
+  headingVariant?: TextProps['variant']
+  text: string
+  tags?: CategoryCardTag[]
+  tagOptions?: Pick<TagProps, 'hyphenate' | 'truncate' | 'textLeft'>
+  href?: string
+  colorScheme?: 'blue' | 'purple' | 'red'
+  /** The heading above is truncated instead of overflowing */
+  truncateHeading?: TextProps['truncate']
+  /** Use event listener to check wether or not to place the image below the text content */
+  autoStack?: boolean
+  /** The card width breakpoint that the image should stack below content when autoStack = true */
+  stackWidth?: number
+  /** Hyphenate the heading */
+  hyphenate?: boolean
+  to?: string
+
+  //TODO: DOCUMENT THIS
```

Comment:
Has this been documented like the TODO says?

Reply:
No, but since it's only used here, it hasn't been a priority, but will be done